### DISCUSSION TITLE
Silent power supply

### DISCUSSION THREAD

**Discussion by Tiwatz** [xblax/flashforge_ad5m_klipper_mod#132] [Ideas]
This is not related to the mod but maybe someone else wants to try this.
I added a KSD9700 NO Thermal Control Switch in series with my psu fan so that the fan turns off when the temperature is low enough. I chose the 40 deg one, seems reasonable. Then I used thermal glue to attach it between the fins of a large heat sink inside the psu.  I have tried it for a couple of prints now and the fan seems to start just after a few minutes printing.
https://www.aliexpress.com/i/32801427417.html 

Edit: I attached some photos

![20240406_092127](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/162510297/d0e71f74-0f90-4903-a718-7963cea701ca)
Unscrew these 4 screws to remove psu from the printer.

![20240406_092318](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/162510297/375eb16a-cc27-42c1-90e9-ef85d1520843)
Remove 4 screws at top of psu. 2 in photo and 2 on the other side

![20240406_092506](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/162510297/ecb6b67f-4cb5-4782-9ceb-b66c92d7b178)

![20240406_092526](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/162510297/23468a67-8cbf-4fe4-886c-88c8ca1bea9c)

**Comment by consp** [comment]
@g992 maybe one for your repo as well?

**Comment by xblax** [comment]
That's a nice mod. I replaced the whole power supply with a genuine Meanwell. If I knew these kind of switches existed I would have done that instead. Maybe I was not searching for the right term.

**Comment by Tiwatz** [comment]
I also noticed that the fan was running at different speeds depending on what I guess is the load on the psu so maybe there is something that can be done with the regulation circuit to make it silent when there is no load. So in that case we don't have to add an extra switch. But my electronics knowledge is not good enough to revert engineer this.

**Comment by b-fitzpatrick** [comment]
I replaced my PS with a Meanwell. Being silent when idle is great, but the Meanwell fan is louder than the stock PS when running. I noticed that the fan opening on the Meanwell looked smaller, but I didn't open it to check the size of the fan.

I've done quite a bit to make the printer quieter overall, and the difference probably wouldn't be noticeable in stock config. Anyway, I opened the stock PS to look into these thermal switches. It's quite a bit different than Tiwatz's. Notably, the fan is 24V instead of 12V.

I'm not sure I'm going to do anything, but I thought I'd post in case someone else is looking.

![image](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/9221473/34dd2418-fba1-4ec3-8ce1-c18bb9d4b9a7)

**Reply by consp** [reply]
Looks like a 40 or 60mm fan, you could always replace it with a noctua or other silent one :) (noctual has 24v versions of almost everything).

>  the fan is 24V

Which is to be expected on a 24V PSU. Why bother with adding another stepdown stage when 24V fans exist.

**Reply by b-fitzpatrick** [reply]
It's a 6015 fan. I did check for a noctua, but their 60 mm fan is 25 mm thick and won't fit.

I guess I thought it was more interesting that Tiwatz's fan was 12V. I opened the MW this morning. I was thinking maybe I could transfer the fan from the stock PS to the MW. Turns out, the MW also has a 12V fan. It appears to be the 12V version of the same fan that's in the stock PS. I tried swapping covers, and that almost worked, but there were minor but significant differences. Instead, I enlarged the fan opening of the MW cover. About to see if that made any difference.

Here's the Mean Well LRS-350-24:
![image](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/9221473/2ff339e4-5e85-49bf-9c7c-37aee382420c)

Here's a side-by-side showing the difference in fan openings:
![image](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/9221473/af1e670c-19b1-46b2-a04d-4005173bdd9a)

And here's a post-op:
![image](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/9221473/e8168137-9d4d-4464-900b-45897a544b3f)

**Reply by b-fitzpatrick** [reply]
There was no noticeable improvement from enlarging the fan opening on the MW PS. I ended up swapping back to the stock PS for now, as the fan noise is lower when running. I'm getting pretty quick at swapping ;-).

I think I'll order a thermal switch. Any suggestions on where to mount it? There's only one small heatsink in the stock PS, and there's something (possibly another thermal switch) already mounted to it. I could mount to the same one.

**Reply by consp** [reply]
> 6015 

Ah that's a less available one, too bad. You could always print a new enclosure ;)

**Reply by b-fitzpatrick** [reply]
Perhaps. I'm not sure if there's enough room between the PS and the back panel to allow the fan to stick out an additional 10 mm.

**Reply by ffontanac** [reply]
> I think I'll order a thermal switch. Any suggestions on where to mount it? There's only one small heatsink in the stock PS, and there's something (possibly another thermal switch) already mounted to it. I could mount to the same one.

I have the same PSU and just installed the thermal switch in the same heatsink the actual thermal switch  is (wich works in the oposite way. It's a Normally closed switch and switchs off the PSU when it reaches 115ºC  to prevent fire hazard)

The only problem is that we have hot summers and once the switch is activated it takes hours to deactivate  after the print finish (or even never deactivate in a hot day). Probably a switch with 45-50ºC trigger would be better.

**Comment by b-fitzpatrick** [comment]
I bought a second non-Pro, and it must have a different power supply. I haven't cracked it open yet, but it's silent with no load. As soon as a heater turns on, the fan kicks on. Just did a PID tune, and the fan cycled on and off with the heater. Unfortunately, it's louder; sounds about like the Mean Well.

**Reply by consp** [reply]
> but it's silent with no load

Mine has that as well and is not that silent. Since I mostly run with the internal blower on I don't mind it but at least it sounds the same as yours.

**Reply by b-fitzpatrick** [reply]
For completeness, here's the power supply from this one:
![image](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/9221473/83675650-b0eb-47b4-a4db-a2264d3121dc)

**Comment by b-fitzpatrick** [comment]
I ended up putting Mean Well PSUs in my AD5Ms. The fan spins up at 50C, but I found the fan too loud. I ended up replacing the 12V fan with a 24V fan, and as you'd expect, it's very quiet running at half-voltage. I assume the printer isn't pulling anywhere near the rated current, so I expect it will be sufficient.

**Comment by DougJoseph** [comment]
My PS is like @b-fitzpatrick 's, not like @Tiwatz 's, and my fan also is 24v not 12v. 

Note that it does not have the same aluminum fin structure thing that @Tiwatz 's fan has, so I'm unsure how to proceed. I have the thermostatic switches, but confused now about where to install the business end of it. Here are pics:

![IMG_0781](https://github.com/user-attachments/assets/9111523e-c62c-473a-8db0-8a96e0d63f0e)

![IMG_0782](https://github.com/user-attachments/assets/e62fc7df-47ff-4c0f-b4d1-64af63e81ba8)

![IMG_0783](https://github.com/user-attachments/assets/eaadf83c-2b4c-4f14-837d-0685fa6b7050)

![IMG_0784](https://github.com/user-attachments/assets/8baf771b-db71-4181-87cc-521f9f885494)

![IMG_0785](https://github.com/user-attachments/assets/63a443a5-2f70-407b-bcc7-ee6b7d00542a)

**Comment by DougJoseph** [comment]
OK, so I did something I was reasonably certain would not hurt and would likely help. I removed the cover, removed the 24v fan from the cover, and did surgery to take out the metal forming a "screen" in front of the fan blades. I have 3 AD5Ms and only one of them had a PS fan that had an ear-piercing frequency to it. Taking away the metal screen in front of the fan totally solved that. The hum of the fan is not only now quieter, but the frequency of it is no longer ear piercing. See pics. Your mileage may vary. Sound is rather subjective. I'm convinced this helped me. Fan still runs constantly but now I can stand it.

**BEFORE:**

![IMG_0786](https://github.com/user-attachments/assets/9f3bcfd3-e60f-47e8-857a-e1460dca19ef)

**AFTER:**

![IMG_0787](https://github.com/user-attachments/assets/677a6f8e-0d51-4014-9c86-3196270e2d4f)

**And I also got this nice Kungfu Throwing Star as a bonus:**

![IMG_0788](https://github.com/user-attachments/assets/75614387-1701-4aff-989c-9f884d151b49)

**Back in place:**

![IMG_0789](https://github.com/user-attachments/assets/b566ef77-6e4a-4110-b009-9772f9715c63)
